src/misc.cpp

```diff
@@ -363,14 +363,15 @@ void std_aligned_free(void* ptr) {
 
 #if defined(__linux__) && !defined(__ANDROID__)
```

Comment:
probably if we use std_aligned_alloc, we can remove the `!defined(__ANDROID__)` flag. Not sure what was the reason to exclude android at that point. We can now cross-compile rather easily to test.

Reply:
The reason was madvise(), IIRC. But that got #ifdef'd anyway for some ancient kernels. So, I think we could use the same code paths.